src/drawing/DrawingAnnotations.tsx

```diff
@@ -119,7 +121,12 @@ const DrawingAnnotations = (props: Props): JSX.Element => {
             )}
 
             {isCreating && (
-                <DrawingCreator className="ba-DrawingAnnotations-creator" onStart={handleStart} onStop={handleStop} />
+                <DrawingCreator
+                    className="ba-DrawingAnnotations-creator"
+                    onStart={handleStart}
+                    onStop={handleStop}
+                    stroke={{ ...defaultStroke, color }}
```

Comment:
I'm wondering if it would be better to just pass in `color` as a prop and let `DrawingCreator` assemble the `stroke` object for subsequent use in child components